db/schema.rb

```diff
@@ -41,6 +41,7 @@
     t.datetime "concluded_at"
     t.datetime "created_at", null: false
     t.integer "established_id"
+    t.datetime "follow_up_mail_sent_at"
```

Comment:
Not sure if this is the best way to track this, but we need a way to fetch matches older than 1 month that we haven't processed yet...